M2/Macaulay2/packages/Resultants.m2

```diff
@@ -1133,7 +1141,7 @@ document {
     Key => {fromPluckerToStiefel,(fromPluckerToStiefel,RingElement),(fromPluckerToStiefel,Matrix),(fromPluckerToStiefel,Ideal)}, 
     Headline => "convert from Plucker coordinates to Stiefel coordinates", 
     Usage => "fromPluckerToStiefel f", 
-    Inputs => { "f" => RingElement => {" or a ",TO "Matrix", ", or an ",TO "Ideal",", in the coordinate ring of a Grassmannian ",TEX///$\mathbb{G}(k,n)$///}}, 
+    Inputs => { "f" => RingElement => {" or a ",TO2{Matrix,"matrix"}, ", or an ",TO2{Ideal,"ideal"},", in the coordinate ring of a Grassmannian ",TEX///$\mathbb{G}(k,n)$///}}, 
```

Comment:
By the way, this is exactly what `ofClass` is for.

Reply:
Ah, ok. Thanks!